project/open-source-strategy.md

```diff
@@ -54,54 +50,37 @@ Core and starting point for the open source project is the development of the so
+If you work on private repositories to get something started before it gets published work on them like they would already be public in terms of what you put in there so that they can smoothly be published once they are ready.
 
 #### Contributing guidelines
 
 The [CONTRIBUTING.md](https://github.com/dtr-org/unit-e/blob/master/CONTRIBUTING.md) file is the central place to document the development process. There is a [great template](https://github.com/nayafia/contributing-template) for what should be covered there.
 
 The CONTRIBUTING.md file should cover all the following sections of the development process.
 
-##### Actions
-
-* [Refine CONTRIBUTING.md](https://github.com/dtr-org/unit-e-docs/issues/22)
-
 #### Tracking work
 
 Work is tracked in GitHub [issues](https://github.com/dtr-org/unit-e/issues) and [pull requests](https://github.com/dtr-org/unit-e/pulls). Keeping discussions there (or at least referencing them from there) keeps them close to the code and creates transparency.
 
 GitHub [milestones](https://github.com/dtr-org/unit-e/milestones) are used to group and schedule work and [labels](https://github.com/dtr-org/unit-e/labels) to classify issues.
 
-There are templates for issues and pull request which help to make sure that the quality of new ones is high and required information is there. This is especially useful for new and external contributors but can also serve as a reminder and guideline for everybody.
-
-##### Actions
-
-* [Create issue template](https://github.com/dtr-org/unit-e-docs/issues/23)
-* [Create PR template](https://github.com/dtr-org/unit-e-docs/issues/24)
-* [Define issue labels](https://github.com/dtr-org/unit-e-docs/issues/25)
-* [Define milestones](https://github.com/dtr-org/unit-e-docs/issues/26)
+There are templates for [issues](https://github.com/dtr-org/unit-e/tree/master/.github/ISSUE_TEMPLATE) and [pull requests](https://github.com/dtr-org/unit-e/blob/master/.github/PULL_REQUEST_TEMPLATE.md) which help to make sure that the quality of new ones is high and required information is there. This is especially useful for new and external contributors but can also serve as a reminder and guideline for everybody.
 
 #### Architectural decisions
 
-Architectural decisions are taken and documented using the [Architecture Decision Record](https://github.com/dtr-org/unit-e-docs/tree/master/adrs) framework. See [ADR-1](https://github.com/dtr-org/unit-e-docs/blob/master/adrs/2018-08-16-ADR-1%20Adopt%20ADR.md) for more details.
-
-##### Actions
-
-* [Decide on scope and visibility of ADRs](https://github.com/dtr-org/unit-e-docs/issues/27)
+Architectural decisions are taken and documented using the [Architecture Decision Record](https://github.com/dtr-org/unit-e-project/tree/master/adrs) framework. See [ADR-1](https://github.com/dtr-org/unit-e-project/blob/master/adrs/2018-08-16-ADR-1%20Adopt%20ADR.md) for more details.
```

Comment:
The link to ADR-1 seems to be broken (404 for me). Maybe it should be https://github.com/dtr-org/unit-e-project/blob/master/adrs/ADR-0001.md#adr-1-adopt-adr

Reply:
Will fix it.